corva/models/base.py

```diff
@@ -34,6 +27,13 @@ class CorvaGenericModel(GenericModel):
     Config = CorvaModelConfig
 
 
+class BaseEvent(CorvaBaseModel, ABC):
```

Comment:
can we move `Config` attribute to the `BaseEvent` and get rid of multiple inheritances here?

Reply:
What is the problem with inheriting from CorvaBaseModel? It is meant to be a base class for every model

Reply:
But we have only one class which inherits it, right?

Reply:
there are 5 classes that inherit from CorvaBaseModel and only 1 that inherits from CorvaGenericModel, so maybe we should get rid of the latter

Reply:
deleted CorvaGenericModel

Reply:
perfect!